.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+
+jobs:
+  run-tests:
+    strategy:
+      matrix:
+        include:
```

Comment:
Add ubuntu-24.04-arm + windows-11-arm runner

Reply:
On ``windows-11-arm`` with ``clang-cl``, I get this error: ``_EMIT_STL_ERROR(STL1000, "Unexpected compiler version, expected Clang 18.0.0 or newer.");``.

This is because the runner have the version 16.0.6 of clang-cl :/